.env.example

```diff
@@ -21,6 +21,8 @@ PEM_FILE_PATH=/absolute/path/to/your/aws-key.pem# absolute path pointing to the
 # Polygon network based variables (see configs/README.md) for more detailed info
 DEFAULT_STAKE=10000# default stake for each validator (in matic)
 DEFAULT_FEE=2000# default amount of fee to topup heimdall validator
+NETWORK= # mention "mumbai", "mainnet" or leave empty if spinning up a local devnet
+HEIMDALL_SEEDS= # Heimdall seeds to be added in config.toml (when running a mainnet/mumbai node )
```

Comment:
can we add an example here?

Reply:
Done: [88a5694](https://github.com/maticnetwork/matic-cli/pull/157/commits/88a56943bc7a38b12a738340a89a6db70e4bd69f)